userspace/libsinsp/sinsp.cpp

```diff
@@ -1606,6 +1606,23 @@ void sinsp::set_snaplen(uint32_t snaplen)
 	}
 }
 
+void sinsp::set_fullcapture_port_range(uint16_t range_start, uint16_t range_end)
+{
+	//
+	// If set_snaplen is called before opening of the inspector,
+	// we register the value to be set after its initialization.
+	//
+	if(m_h == NULL)
+	{
+		throw sinsp_exception("set_fullcapture_port_range called before capture start");
+	}
+
+	if(is_live() && scap_set_fullcapture_port_range(m_h, range_start, range_end) != SCAP_SUCCESS)
```

Comment:
If we call this function on an offline inspector we don't get an error, the call silently ignores everything, is that expected?

Perhaps `is_live()` could be moved to a separate check and throw an exception if false, or removed altogether and let scap return error if opened on a trace file.